.github/workflows/package_deploy.yml

```diff
@@ -0,0 +1,46 @@
+name: Package and Deploy
+
+on:
+  push:
+    branches:
+      - feature/package-artifact
+  workflow_dispatch:
```

Comment:
this is cool!
but does it need the `:`?
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch

Reply:
I've always had it, I guess it's a force of habit. It's also how Copilot suggested it. 🤷‍♂️